sample/HttpTrigger/index.js

```diff
@@ -7,9 +7,17 @@
     }
 
     if (typeof req.query.name == 'undefined') {
-        context.done(null, "Please pass a name on the query string");
+        context.res = {
+            status: 400,
+            body: "Please pass a name on the query string"
+        };
     }
     else {
-        context.done(null, "Hello " + req.query.name);
+        context.res = {
+            status: 200,
+            body: "Hello " + req.query.name
+        };
```

Comment:
Would be good to also show how to set the content type, since that's a pretty key thing in http scenarios.